bpdm-bridge-dummy/src/main/resources/application.properties

```diff
@@ -32,6 +32,7 @@ bpdm.bridge.query-page-size=100
 # Connection to Pool and Gate
 bpdm.pool.base-url=http://localhost:8080/
 bpdm.gate.base-url=http://localhost:8081/
+bpdm.bridge.syncAuthorities=sync_company_data
```

Comment:
The permission names belong to the auth properties/profile. Missing permission names should have no effect when auth is disabled